src/modules/fetch.js

```diff
@@ -0,0 +1,25 @@
+import toCelsius from './converter';
+
+async function getWeather(chosenCity) {
```

Comment:
- [ ] You are asked to use `ES6` syntax in this project so you should be using `arrow functions` here instead of the `function keyword`